### PR TITLE
changes for search and UI navigation

### PR DESCRIPTION
**Pull request by PhilSharland**

### CODE REVIEW

File: routes/fieldforce.rb [hunk 1]
```diff
@@ -30,23 +35,58 @@ def url_format(str)
 # Get Individual Field Worker Details
 get '/fieldforce/:fieldworkerid' do |fieldworkerid|
   authenticate!
-  fieldworker = []
+  fieldworkerdetails = []
```
Comment (johntopleyons) [inline routes/fieldforce.rb]: Your variable names are a bit inconsistent. `separate_words_with_underscores`.

File: routes/home.rb [hunk 2]
```diff
@@ -140,7 +140,7 @@ def h(text)
     multi_query_flag = true
   end
 
-  puts 'http://' + CENSUS_FSDR_HOST + ':' + CENSUS_FSDR_PORT + "/fieldforce/#{search_params}"
+  puts search_params
```
Comment (johntopleyons) [inline routes/home.rb]: Should this be committed?

File: views/field_worker.erb [hunk 3]
```diff
@@ -1,155 +1,29 @@
 <%= erb :menu, locals: { pageid: 'home' } %>
 
-<% if fieldworker.any? %>
+<% if !fieldworkerdetails.nil? %>
```
Comment (johntopleyons) [inline views/field_worker.erb]: Don't forget that you can do `<% unless fieldworkdetails.nil? %>`, which is easier to read.

File: views/searchresults.erb [hunk 4]
```diff
@@ -17,8 +17,8 @@
   <tbody>
     <% results.each_with_index do |result| %>
       <tr>
-        <% if result['id'].present? %>
-          <td><%=h result['id'] %></td>
+        <% if result['uniqueEmployeeId'].present? %>
```
Comment (johntopleyons) [inline views/searchresults.erb]: Variable naming again! Should be `unique_employee_id`.

File: routes/home.rb (hunk 2 above)
Comment (PhilSharland) [reply routes/home.rb]: oops :)

File: views/searchresults.erb (hunk 4 above)
Comment (PhilSharland) [reply views/searchresults.erb]: That's in the response from the endpoint. so it's what I get. agreed that other naming standards have been wooly and changed appropriately.

**Pull request merged**